Makefile

```diff
@@ -1,6 +1,7 @@
 # -*- MakeFile -*-
 
 CC = clang -std=c89
+CCTEST = clang -std=c99 -lcriterion
```

Comment:
Is `CCTEST = $(CC) -lcriterion` possible?

Reply:
I couldn't think of a better way. `-std=c99` is mandatory.

See https://github.com/Snaipe/Criterion: 

>  C99 and C++11 compatible.

Reply:
Ok, great.